test/unit/initialize.spec.ts

```diff
@@ -555,7 +556,9 @@ describe("Initialization", () => {
             const endIndex = flicking.getIndex();
             const indexAtDestPos = Math.floor(destPos / panelWidth);
             // As all delta is above threshold, it should change panel at least once
-            const expectedIndex = Math.min(nearestPanel.getIndex() + snapCount, Math.max(1, indexAtDestPos));
+            const expectedIndex = eventDelta >= 50
```

Comment:
@WoodNeck It would be better to comment for this.

Reply:
That's right, I forgot to add one. Thanks!